azure.gemspec

```diff
@@ -35,8 +35,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency('azure-core',              '~> 0.1')
   s.add_runtime_dependency('faraday',                 '~> 0.9')
   s.add_runtime_dependency('faraday_middleware',      '~> 0.10')
-  s.add_runtime_dependency('json',                    '~> 1.8')
-  s.add_runtime_dependency('mime-types',              ['>= 1', '< 3.0'])  # vagrant-share and other stuff relies on 1
+  s.add_runtime_dependency('mime-types',              ['>= 1', '< 4.0'])  # vagrant-share and other stuff relies on 1
```

Comment:
@sarangan12 & @veronicagg Did you guys get chance to talk about what were the breaking changes between 2.0 and 3.0 of mime-type gem and whether are we wrapping those apis or exposing those through our gem?